spec/support/feature_spec_helper.rb

```diff
@@ -10,4 +10,11 @@ def login_as(user)
     setup_mock_auth(:myusa, user)
     visit '/auth/myusa'
   end
+
+  # adapted from http://stackoverflow.com/a/25047358
+  def fill_in_selectized(key, *values)
```

Comment:
let's put this in the `observer_spec` file as a method inside the `describe` block so it's scoped. Easier to see what's going on that way!